run_service.sh

```diff
@@ -175,6 +175,92 @@ get_on_chain_service_state() {
     echo "$state"
 }
 
+store=".trader_runner"
+rpc_path="$store/rpc.txt"
+operator_keys_file="$store/operator_keys.json"
+keys_json="keys.json"
+keys_json_path="$store/$keys_json"
+agent_address_path="$store/agent_address.txt"
+service_id_path="$store/service_id.txt"
+service_safe_address_path="$store/service_safe_address.txt"
+store_readme_path="$store/README.txt"
+
+# Function to create the .trader_runner storage
+create_storage() {
+    echo "This is the first run of the script. The script will generate new operator and agent instance addresses."
+    echo ""
+    # Generate the operator's key
+    address_start_position=17
+    pkey_start_position=21
+    poetry run autonomy generate-key -n1 ethereum
+    mv "$keys_json" "../$keys_json_path"
+    operator_address=$(sed -n 3p "../$keys_json_path")
```

Comment:
I created a couple of functions to extract private key and address from keys.json files. Try using them.

Reply:
This should be done on a separate PR. This one just moves existing stuff in the new method, the suggested refactoring should be performed separately.